snappy/kernel_os.go

```diff
@@ -65,7 +65,7 @@ func copyAll(src, dst string) error {
 // extractKernelAssets extracts kernel/initrd/dtb data from the given
 // Snap to a versionized bootloader directory so that the bootloader
 // can use it.
-func extractKernelAssets(s *snap.Info, snapf snap.File, flags InstallFlags, inter progress.Meter) error {
+func extractKernelAssets(s *snap.Info, snapf snap.Container, flags InstallFlags, inter progress.Meter) error {
```

Comment:
snapf is not actually used anymore in this function I think (see previous comment)